data/_ui/interfaces.txt

```diff
@@ -266,7 +266,7 @@ interface "main menu"
 		dimensions 90 30
 	
 	visible
-	button l "_Load / Save..."
+	button l "Manage Pi_lots..."
```

Comment:
@TheGiraffe3 -- what do you think about these ellipses?  They were already there, so I left them.  But hey, we're here changing stuff, so...

Reply:
I think it would be better to remove them.
At the moment they don't mean anything, only implying that there is more then what is on the button while there really isn't.
Actually, those ellipses are probably there because the original label also allowed for creating new pilots but didn't imply that in its name. Now, though...

Reply:
Well, I'm going to remove them.  But I'm also going to point out that "Preferences..." is now the only thing with an ellipses and based on your logic so far, I'm thinking that ought to change too.

Reply:
The ellipses indicates that the button goes somewhere else; a new panel. I don't think it's any less valid on "Manage Pilots" than it was on "Load / Save" or "Preferences".

Reply:
Then why doesn't "Enter Ship" have ellipses?
It completely changes the scene.

Reply:
Alright, I'm gonna wait for a tie-breaker vote before I commit anything more.  I don't have strong feelings about the ellipses.

Reply:
Seconding warp's thoughts. If it turns out we don't like it later, we can change it, but I think keeping the ellipses for the button is fine.

Reply:
These have been added back.